contracts/PositionManager.sol

```diff
@@ -123,363 +139,60 @@ contract PositionManager is IPositionManager, ERC721Holder {
         return uniswapNFTsMemory;
     }
 
+    function toggleModule(
+        uint256 tokenId,
+        address moduleAddress,
+        bool activated
+    ) external override onlyOwner {
+        activatedModules[tokenId][moduleAddress] = activated;
+    }
+
```

Comment:
Are we sure that toggle is the best way to keep track of the state? Maybe enable/disable is better?
Also, we should add NatSpec